ssl/ssl_local.h

```diff
@@ -1760,6 +1760,7 @@ typedef struct sigalg_lookup_st {
 
 typedef struct tls_group_info_st {
     int nid;                    /* Curve NID */
+    const char *keyname;
```

Comment:
We're consistently calling it "keytype" elsewhere, aren't we?

Reply:
Are we? Ok. I'll change.